hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsQueueResource.kt

```diff
@@ -25,6 +25,7 @@ class HmppsQueueResource(hmppsQueueService: HmppsQueueService) {
    * See test-app/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstemplatepackagename/config/ResourceServerConfiguration.kt for Spring Security config.
    * See https://github.com/ministryofjustice/hmpps-helm-charts/blob/main/charts/generic-service/templates/retry-dlq-cronjob.yaml and test-app/helm_deploy/hmpps-template-kotlin/example/ingress.yaml for Kubernetes config.
    */
+  @Deprecated("Please use the errorVisibilityTimeout configuration available in the configuration properties.")
```

Comment:
Is Deprecated a bit harsh ? You might have rare transient errors that you are happy to retry every 10 minutes? Would documentation be a better way of conveying the advantages of the errorVisibility approach?

Reply:
That's fair. Not quite sure where the documentation should live... actually probably in the tech docs, which I'll need to update when this is all stable.